DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -140,11 +146,47 @@ BulkAdminFieldAnswersViewModel model
             );
         }
 
+        [HttpGet]
+        [Route("{customisationId:int}/AdminFields/{promptNumber:int}/Remove")]
+        public IActionResult RemoveAdminField(int customisationId, int promptNumber)
+        {
+            var answerCount =
+                userDataService.GetAnswerCountForCourseAdminField(customisationId, promptNumber);
+
+            if (answerCount == 0)
+            {
+                return RemoveAdminFieldAndRedirect(customisationId, promptNumber);
+            }
+
+            var promptName =
+                courseAdminFieldsService.GetPromptNameForCourseAndPromptNumber(customisationId, promptNumber);

```

Comment:
Could this be just "GetPromptName" - it should be clear from the params what we're using to get it, and it will make this easier to read :)

Reply:
I agree it's a long and confusing name, changed it